cmd/operator-sdk/add/api.go

```diff
@@ -69,6 +80,53 @@ Example:
 	$ tree deploy/crds
 	├── deploy/crds/app.example.com_v1alpha1_appservice_cr.yaml
 	├── deploy/crds/app.example.com_appservices_crd.yaml
+
+Ansible Example:
+
+	$ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=Mykind	
+	$ tree ./roles
+	./roles
+		├── memcached
+		│   ├── README.md
+		│   ├── defaults
+		│   │   └── main.yml
+		│   ├── tasks
+		│   │   └── main.yml
+		├── mykind
+		│   ├── README.md
+		│   ├── defaults
+		│   │   └── main.yml
+		│   ├── tasks
+		│   │   └── main.yml
+Helm Example:
+	
+	$ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=Mongodb --helm-chart=stable/mongodb
```

Comment:
I think we need to be careful which examples we use because we tell users not to put unrelated APIs in the same operator.

Not sure I can think of any examples of the top-of my head, but is there something related to memcached that would make sense to always deploy alongside memcached? Maybe a memecached frontend UI app or a memcached monitoring app?

Reply:
For Helm, it also might be nice to include other examples [similar to the ones defined in the `new` cmd](https://github.com/operator-framework/operator-sdk/blob/179d8d6c81f8a14ff386f7d71372859551f56162/cmd/operator-sdk/new/cmd.go#L67-L97).